README.md

```diff
@@ -120,3 +120,152 @@ the components involved in the test:
 ``
 cd test/integration && dlv test -- -test.run ^TestCrud$
 ``
+
+#### E2E
+
+The fnord E2E tests can run in an unmanaged, managed, or hybrid modes. For both
```

Comment:
We should probably change this to federation-v2

Reply:
Fixed.